src/request/browser.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Make a request using `XMLHttpRequest`.
+ *
+ * @param   {Promise} Promise
+ * @param   {String}  method
+ * @param   {String}  url
+ * @param   {String}  body
+ * @param   {Object}  headers
+ * @returns {Promise}
+ */
+module.exports = function request (Promise, method, url, body, headers) {
+  return new Promise(function (resolve, reject) {
+    var xhr = new window.XMLHttpRequest()
+
+    xhr.open(method, url)
+
+    xhr.onload = function () {
+      return resolve({
+        status: xhr.status,
+        body: xhr.responseText
```

Comment:
I just killed the headers two minutes ago. I don't need them. I mentioned this in my previous comments.